notebooks/15 Pipelining Estimators.ipynb

```diff
@@ -160,7 +160,7 @@
     "This will then be fed to the ``fit`` of the next step, and so on.\n",
     "Finally, on the last step, only ``fit`` is called.\n",
     "\n",
-    "![pipeline](figures/pipeline.svg)\n",
```

Comment:
nbconvert should be able to convert svg to pdf for latex compilation. If not, that's an issue in nbconvert - though thinking about it, there might have been issues last time I tried. Maybe that's only true for python generated figures, not embedded ones. I think the only way to make this render in both is to use PNG :-/